src/Microsoft.Health.Fhir.Core/Features/Operations/Export/ExportJobTask.cs

```diff
@@ -211,16 +208,33 @@ private async Task UpdateJobRecordAsync(CancellationToken cancellationToken)
             }
         }
 
-        // Get destination info from secret store, create appropriate export client and connect to destination.
+        // Get destination information, create appropriate export client and connect to destination.
```

Comment:
I don't think we want to aquire token on every loop right? The token is valid for certain period of time, so we should be able to reuse it until we have to refresh it?

Reply:
Also, i don't think it's the export job task's responsibility to acquire access token.

Reply:
The original change had it only getting the token at the beginning of the export process. Once we connect, we get into the while loop for processing the export. Like you mentioned, I have moved the  acquiring access token logic out of the `ExportJobTask` class. Now that logic is abstracted away as part of the `IExportDestinationClient`